Vishal/add staking faqs (#222)
onflow/flow: docs/content/staking/faq.mdx

---
title: Staking FAQ
sidebar_title: FAQs
description: Frequently Asked Questions
---

### Where will users receive their staking reward for each staking option?

If you staked using Flow Port, then you should be able to see your staking rewards there. You can also withdraw the rewards or manually re-stake them through Flow-Port.

If you staked using a staking provider such as Kraken, Blocto or Finoa, they should be able to provide you more details on how to receive the staking reward.

### Will staking rewards be automatically re-staked?

There will be *no* automatic re-staking of staking rewards with Flow Port (ledger/blocto).

If you staked using a staking provider such as Kraken, Blocto or Finoa, they should be able to provide you more details.

### Can a validator change its fees? Would the Account Key contract need to stay in Beta permanently to enable the validator to change the fee percentage? Does this change take effect immediately or at the next epoch?

> For the beta period we’re enforcing a minimum fee of 10%.

Reviewer: ```suggestion
For the beta period we’re enforcing a minimum delegation fee of 8%.
```

> In the future it will be a parameter you can update.

Reviewer: ```suggestion
```
The current understanding is that they won't be able to update their fee

> The contract includes a function you can call to update if you would like but this wouldn’t require the contract to be in beta.

Reviewer: ```suggestion
```

Rewards are only paid out every epoch, so the fee at the time of the epoch payout would be the one applied.

### Can a token holder stake to multiple nodes? If yes, how is the stake split between them?

A token holder will be able to stake multiple nodes in the future.

> However, currently there is a restriction in place such that there is a 1-to-1 relationship between an account address and an active stake.

Reviewer: ```suggestion
However, currently there is a restriction in place from the token lockup rules such that there is a 1-to-1 relationship between an account address and an active stake.
```
